src/main/java/buildcraftAdditions/blocks/BlockBackpackStand.java

```diff
@@ -85,4 +86,18 @@ public void breakBlock(World world, int x, int y, int z, Block block, int meta)
 		}
 		super.breakBlock(world, x, y, z, block, meta);
 	}
+
+	@Override
+	public boolean canPlaceBlockAt(World world, int x, int y, int z) {
+		return super.canPlaceBlockAt(world, x, y, z) && world.isAirBlock(x, y + 1, z);
+	}
+
+	@Override
+	public void onNeighborBlockChange(World world, int x, int y, int z, Block block) {
+		if (block == ItemsAndBlocks.backpackStandGhost) {
```

Comment:
this will cause issues, if it's a ghost block that is adjecent but not on top of the backpackstand it will also triger